snakemake_executor_plugin_googlebatch/__init__.py

```diff
@@ -155,6 +155,15 @@ class ExecutorSettings(ExecutorSettingsBase):
         },
     )
 
+    service_account: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": "The email of a customer compute service account",
+            "env_var": False,
```

Comment:
I would perhaps enable env_var True here, because might the ID be considered more private that an equivalent variable?